src/check/property/AsyncProperty.ts

```diff
@@ -23,9 +22,8 @@ function asyncProperty<Ts extends [unknown, ...unknown[]]>(
   }
   const arbs = args.slice(0, args.length - 1) as { [K in keyof Ts]: Arbitrary<Ts[K]> };
   const p = args[args.length - 1] as (...args: Ts) => Promise<boolean | void>;
-  const mappedArbs = arbs.map((arb): typeof arb =>
-    convertFromNext(new AlwaysShrinkableArbitrary(convertToNext(arb)))
-  ) as typeof arbs;
+  arbs.forEach(assertIsArbitrary);
```

Comment:
Not sure we want to assert from that place 🤔
We used to assert from `tuple` but I don't know if it was better or not

Reply:
Let's keep the check here for the moment!